### ISSUE TITLE
[macOS] VSCode terminal throwing an error after choosing `remark: beautify markdown code`

### ISSUE THREAD

**Issue by dphov** [mrmlnc/vscode-remark#5]
VSCode 1.8.1
macOS 10.12.2

```sh
[Remark]
TypeError: Cannot read property 'length' of undefined
```

DevConsole
```
Illegal value for `line`: Error: Illegal value for `line`
    at t.lineAt (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:11:2702)
    at Object.lineAt (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:11:1706)
    at GuidesController.updateSelection (/.vscode/extensions/spywhere.guides-0.6.1/out/extension.js:41:51)
    at e.invoke (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:4:31992)
    at e.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:5:8945)
    at t.$acceptSelectionsChanged (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:11:7501)
    at t.e.handle (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:10:4203)
    at s (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:7:11178)
    at p (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:7:11855)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)e.onUnexpectedError @ shell.ts:541(anonymous function) @ shell.ts:457e.onUnexpectedError @ errors.ts:68o @ errors.ts:88t.onUnexpectedExtHostError @ mainThreadErrors.ts:13e.handle @ abstractThreadService.ts:34s @ ipcRemoteCom.ts:269f @ ipcRemoteCom.ts:226_combinedTickCallback @ internal/process/next_tick.js:67_tickCallback @ internal/process/next_tick.js:98
```

**Comment by mrmlnc** [comment]
Hello, @dphov,

You use plugins?

**Comment by dphov** [comment]
Hello, @mrmlnc

Of course I have installed a bunch of plugins

**Comment by mrmlnc** [comment]
You can provide a list of plugins?

Most likely this is related to https://github.com/mrmlnc/vscode-remark#about-error-handling. Try to disable each plugin and run `remark: beautify markdown code`.

**Comment by dphov** [comment]
Alan.MochaSnippets
CoenraadS.bracket-pair-colorizer
DavidAnson.vscode-markdownlint
DotJoshJohnson.xml
EditorConfig.EditorConfig
HookyQR.JSDocTagComplete
IBM.output-colorizer
LaurentTreguier.vscode-simple-icons
NoHomey.theme-realgithub
Orta.vscode-ios-common-files
Orta.vscode-jest
Orta.vscode-react-native-storybooks
Perkovec.require-sort
SirTori.indenticator
TimonVS.ReactSnippetsStandard
TomiTurtiainen.js-complexity-analysis
TwentyChung.jsx
Tyriar.sort-lines
UCL.haskelly
WakaTime.vscode-wakatime
Zignd.html-css-class-completion
abierbaum.vscode-file-peek
adambaldwin.vscode-nsp
akamud.vscode-caniuse
akamud.vscode-javascript-snippet-pack
akamud.vscode-theme-onedark
alechp.react-toolbox-snippets
alefragnani.Bookmarks
alefragnani.project-manager
andradei.minimal-colors
anseki.vscode-color
auiworks.amvim
azemoh.one-monokai
be5invis.vscode-icontheme-nomo-dark
christian-kohler.npm-intellisense
christian-kohler.path-intellisense
cmstead.jsrefactor
cssho.vscode-svgviewer
dai-shi.vscode-es-beautifier
davidbabel.vscode-simpler-icons
dbaeumer.jshint
dbaeumer.vscode-eslint
dbankier.vscode-instant-markdown
donjayamanne.githistory
donjayamanne.javadebugger
donjayamanne.python
dracula-theme.theme-dracula
eg2.vscode-npm-script
emmanuelbeziat.vscode-great-icons
fania.icons-fania
fknop.vscode-npm
formulahendry.auto-close-tag
formulahendry.auto-rename-tag
formulahendry.code-runner
gerane.Theme-Solarized-dark
helixquar.randomeverything
hoovercj.haskell-linter
hoovercj.vscode-ghc-mod
humao.rest-client
idleberg.hopscotch
idleberg.nsis
jakob101.RelativePath
jameshrisho.redux-snippets
jebbs.plantuml
jez9999.vsclassic-icon-theme
joelday.docthis
joshjg.generate-react-component
joshpeng.sublime-babel-vscode
joshpeng.theme-charcoal-oceanicnext
joshpeng.theme-onedark-sublime
joshpeng.theme-tomorrowkit-sublime
jtlowe.vscode-icon-theme
justusadam.language-haskell
kisstkondoros.typelens
kisstkondoros.vscode-codemetrics
lonefy.vscode-JS-CSS-HTML-formatter
mohsen1.prettify-json
monofon.hindent-format
mrmlnc.vscode-attrs-sorter
mrmlnc.vscode-duplicate
mrmlnc.vscode-postcss-sorting
mrmlnc.vscode-remark
mrmlnc.vscode-stylefmt
ms-vscode.Theme-MarkdownKit
ms-vscode.Theme-MaterialKit
ms-vscode.js-atom-grammar
ms-vscode.latex
msjsdiag.debugger-for-chrome
naumovs.color-highlight
paulhoughton.vscode-jscpd
pflannery.vscode-versionlens
phoityne.phoityne-vscode
qinjia.seti-icons
redhat.java
robertohuertasm.vscode-icons
ryu1kn.annotator
samverschueren.yo
seanmcbreen.Spell
shardulm94.trailing-spaces
shinnn.stylelint
spywhere.guides
steve8708.Align
stevencl.addDocComments
sysoev.language-stylus
taichi.react-beautify
teabyii.ayu
timothymclane.react-redux-es6-snippets
tushortz.java-snippets
vector-of-bool.gitflow
veggiemonk.theme-solarized-light-fjs
vigoo.stylish-haskell
vilicvane.es-quotes
vscodevim.vim
vsmobile.vscode-react-native
vstirbu.vscode-mermaid-preview
vuhrmeister.vscode-meteor
wmaurer.change-case
xabikos.JavaScriptSnippets
xabikos.ReactSnippets
ymotongpoo.licenser
zhuangtongfa.Material-theme
ziyasal.vscode-open-in-github

**Comment by mrmlnc** [comment]
Sorry, but I mean Remark plugins (e.g. https://github.com/wooorm/remark/blob/master/doc/plugins.md).

**Comment by dphov** [comment]
Problem solved by installing new version of VSCode 1.9.1

**Comment by mrmlnc** [comment]
Thanks for report 💪
